core/field_input.ts

```diff
@@ -614,6 +615,7 @@ export abstract class FieldInput<T extends InputTypes> extends Field<
       if (target instanceof FieldInput) {
         WidgetDiv.hideIfOwner(this);
         dropDownDiv.hideWithoutAnimation();
+        getFocusManager().focusNode(target);
```

Comment:
Drive by comment: In the case of full-block fields, I think we actually want to focus the parent block for consistency.

Reply:
Didn't know about full-block fields before this. Fortunately, the block being tested here already uses full-block fields in Zelos, so I updated the tests to verify behavior specifically with Zelos and Geras separately. I confirmed that one of the Zelos tests fails without the new `focusNode` line in place.